### PR TITLE
Convert Utc to signed integer

### PR DESCRIPTION
**Pull request by imclerran**
Fix for issue #30, as discussed in Zulip [here](https://roc.zulipchat.com/#narrow/stream/231634-beginners/topic/Code.20review.3F/near/418124387)

### CODE REVIEW

**Review by bhansconnect — DISMISSED** [review]
If your willing, can you add some tests? Specifically would be good to check a a few cases of negative with positive and also a max range test.

**Comment by imclerran** [comment]
Yeah, I can definitely do that!

**Comment by imclerran** [comment]
@bhansconnect How are we looking here? Anything else need tweaking?

File: platform/InternalDateTime.roc [hunk 1]
```diff
@@ -6,7 +6,7 @@ interface InternalDateTime
     ]
     imports []
 
-DateTime : { year : U128, month : U128, day : U128, hours : U128, minutes : U128, seconds : U128 }
+DateTime : { year : I128, month : I128, day : I128, hours : I128, minutes : I128, seconds : I128 }
```
Comment (bhansconnect) [inline platform/InternalDateTime.roc:9]: Not from this PR, but just curious, do we need these to also be I128s, can they be something way smaller?

**Review by bhansconnect — APPROVED** [review]

File: platform/InternalDateTime.roc (hunk 1 above)
Comment (imclerran) [reply platform/InternalDateTime.roc:9]: That thought crossed my mind too... those are huge types for values that will never approach the bounds of their type...
